clouddriver-openstack/src/main/java/org/openstack4j/openstack/image/v2/domain/GlanceImage.java

```diff
@@ -0,0 +1,633 @@
+    return directUrl;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String getSelf() {
+    return self;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String getFile() {
+    return file;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String getSchema() {
+    return schema;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String getRamdiskId() {
+    return ramdiskId;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String getOsDistro() {
+    return osDistro;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String getOsVersion() {
+    return osVersion;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String getKernelId() {
+    return kernelId;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String getInstanceUuid() {
+    return instanceUuid;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String getArchitecture() {
+    return architecture;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public Long getVirtualSize() {
+    return virtualSize;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String getAdditionalPropertyValue(String key) {
+    return additionalProperties.get(key);
+  }
+
+  @JsonAnyGetter
+  public Map<String, String> getAdditionalProperties() {
+    return additionalProperties;
+  }
+
+  @JsonAnySetter
+  public void setAdditionalProperty(String key, String value) {
+    if (key != null && !RESERVED_KEYS.contains(key)) {
+      additionalProperties.put(key, value);
+    }
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public ImageBuilder toBuilder() {
+    return new ImageConcreteBuilder(this);
+  }
+
+  public static ImageBuilder builder() {
+    return new ImageConcreteBuilder();
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String toString() {
+    return MoreObjects.toStringHelper(this)
+      .add("id", id)
+      .add("name", name)
+      .add("tags", tags)
+      .add("imageStatus", status)
+      .add("containerFormat", containerFormat)
+      .add("diskFormat", diskFormat)
+      .add("createdAt", createdAt)
+      .add("updatedAt", updatedAt)
+      .add("minDisk", minDisk)
+      .add("minRam", minRam)
+      .add("isProtected", isProtected)
+      .add("checksum", checksum)
+      .add("owner", owner)
+      .add("visibility", visibility)
+      .add("size", size)
+      .add("locations", locations)
+      .add("directUrl", directUrl)
+      .add("self", self)
+      .add("file", file)
+      .add("schema", schema)
+      .add("architecture", architecture)
+      .add("instanceUuid", instanceUuid)
+      .add("kernelId", kernelId)
+      .add("osVersion", osVersion)
+      .add("osDistro", osDistro)
+      .add("ramdiskId", ramdiskId)
+      .add("virtualSize", virtualSize)
+      .toString();
+  }
+
+
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  public static class Images extends ListResult<GlanceImage> {
```

Comment:
Same deal here with this class and its statics - a lot of boilerplate can be removed in favor of lombok annotations if you are so inclined.